database/sql/src/main/scala/cromwell/database/slick/WorkflowStoreSlickDatabase.scala

```diff
@@ -27,25 +28,24 @@ trait WorkflowStoreSlickDatabase extends WorkflowStoreSqlDatabase {
                                              workflowStateToDelete1: String,
                                              workflowStateToDelete2: String,
                                              workflowStateForUpdate: String)
-                                            (implicit ec: ExecutionContext): Future[(Int, Option[Boolean])] = {
-    val action =  for {
-      restarted <- dataAccess.heartbeatClearedForWorkflowId(workflowExecutionUuid).result.headOption
```

Comment:
This seems to be the only time this `heartbeatClearedForWorkflowId` was used. If so, also kill off the declaration in the Component.